src/components/Charts/deceasedchart.js

```diff
@@ -0,0 +1,137 @@
+import React from 'react';
+import {Bar, defaults} from 'react-chartjs-2';
+
+function DeceasedChart(props) {
+  defaults.global.tooltips.intersect = false;
+  defaults.global.tooltips.mode = 'nearest';
+  defaults.global.tooltips.position = 'average';
+  defaults.global.tooltips.backgroundColor = 'rgba(255, 255, 255, 0.8)';
+  defaults.global.tooltips.displayColors = false;
+  defaults.global.tooltips.borderColor = '#c62828';
+  defaults.global.tooltips.borderWidth = 1;
+  defaults.global.tooltips.titleFontColor = '#000';
+  defaults.global.tooltips.bodyFontColor = '#000';
+  defaults.global.tooltips.caretPadding = 4;
+  defaults.global.tooltips.intersect = false;
+  defaults.global.tooltips.mode = 'nearest';
+  defaults.global.tooltips.position = 'nearest';
+
+  defaults.global.legend.display = true;
+  defaults.global.legend.position = 'bottom';
+
+  defaults.global.hover.intersect = false;
+
+  const ages = [0, 0, 0, 0, 0, 0, 0, 0, 0, 0];
+  let unknown = 0;
+  if (!props.data || props.data.length === 0) {
+    return <div></div>;
+  }
+
+  props.data.forEach((patient) => {
```

Comment:
All these operations can be wrapped under _useCallback_.

Reply:
I think there is no such necessary need to wrap these operations under useCallback.